x-pack/plugin/data-frame/qa/multi-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFrameIntegTestCase.java

```diff
@@ -277,13 +274,13 @@ protected void createReviewsIndex() throws Exception {
             if (i % 50 == 0) {
                 BulkResponse response = restClient.bulk(bulk, RequestOptions.DEFAULT);
                 assertThat(response.buildFailureMessage(), response.hasFailures(), is(false));
-                bulk = new BulkRequest(REVIEWS_INDEX_NAME);
-                day += 1;
+                bulk = new BulkRequest(indexName);
+                day = (day % 18) + 10;
```

Comment:
`day = (day + 1) % 28` would be simpler, it's not clear what pattern is being used here. On the first iteration day = 10, then day = 20, day = 12...